packages/ogmios/test/CardanoNode/mappers.test.ts

```diff
@@ -0,0 +1,26 @@
+import * as mappers from '../../src/CardanoNode/mappers';
+import { EraSummary } from '@cardano-sdk/core';
+import { Schema } from '@cardano-ogmios/client';
+
+describe('cardano node mappers', () => {
+  describe('mapEraSummary', () => {
+    const eraSummary: Schema.EraSummary = {
+      end: { epoch: 102, slot: 13_694_400, time: 44_064_000 },
+      parameters: { epochLength: 432_000, safeZone: 129_600, slotLength: 1 },
+      start: { epoch: 74, slot: 1_598_400, time: 31_968_000 }
+    };
+    test('map ogmios EraSummary to TimeSettings', () => {
```

Comment:
`map ogmios EraSummary to core EraSummary`

Reply:
Done 12e43c9d5d30b08a49deffa127a776b82d0a5a03